lib/rest/client.js

```diff
@@ -99,6 +99,26 @@ function encodePath(_path, _leadingSlash) {
   return path;
 }
 
+function parseResponse(req, res, buffer) {
```

Comment:
Can this function be used by `requestJson()`?

Reply:
`requestJson` just calls `parseJson` which calls this